### PR TITLE
ENG-9835: Policy rule identities field should be omitted by default (v2)

### PR DESCRIPTION
**Pull request by Yowgf**
## Description of the change

Same as #301 , but this is a fix on top of version 2 of the TF provider.

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review

- [x] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [x] Jira issue referenced in commit message and/or PR title

### Testing

Manually tested that the issue does not happen after the changes. Acceptance test TestAccPolicyRuleResource is passing.

### CODE REVIEW

File: cyral/resource_cyral_policy_rule.go [hunk 1]
```diff
@@ -324,19 +327,18 @@ func resourcePolicyRuleUpdate(ctx context.Context, d *schema.ResourceData, m int
...
 
-func resourcePolicyRuleDelete(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	log.Printf("[DEBUG] Init resourcePolicyRuleDelete")
-	c := m.(*client.Client)
-
-	url := fmt.Sprintf("https://%s/v1/policies/%s/rules/%s", c.ControlPlane, d.Get("policy_id").(string), d.Id())
-
-	if _, err := c.DoRequest(url, http.MethodDelete, nil); err != nil {
-		return createError("Unable to delete policy rule", fmt.Sprintf("%v", err))
+func deletePolicyRule() ResourceOperationConfig {
```
Comment (VictorGFM) [inline cyral/resource_cyral_policy_rule.go:330]: Nit: What do you think of naming this function as `policyRuleDeleteConfig`? I think it reflects better its purpose. Just a suggestion though.
```suggestion
func policyRuleDeleteConfig() ResourceOperationConfig {
```

**Review by VictorGFM — APPROVED** [review]
LGTM, just provided a nit comment below. Thanks for this fix @Yowgf!

**Comment by VictorGFM** [comment]
Hey @Yowgf, could you also add the tests you included in [this PR](https://github.com/cyralinc/terraform-provider-cyral/pull/301) to this one?

**Comment by Yowgf** [comment]
> Hey @Yowgf, could you also add the tests you included in [this PR](https://github.com/cyralinc/terraform-provider-cyral/pull/301) to this one?

Done, thanks!

**Pull request merged**